docs/release-notes/Language.md

```diff
@@ -0,0 +1,21 @@
+﻿---
+category: Release Notes
+categoryindex: 600
+index: 2
+title: F# Language
+---
+
+# F\# Language
+
+## Unreleased
```

Comment:
This should specify that it's preview probably, since it makes sense for language version.